src/components/SocialContentWrapper/SocialContentWrapper.jsx

```diff
@@ -1,23 +1,32 @@
 import React from 'react';
 import cx from 'classnames';
 import PropTypes from 'prop-types';
-import { PlaceholderEmbed } from 'react-social-media-embed';
 import './Style.less';
+import config from '@plone/volto/registry';
 
-const checkConsent = (tool) => {
-  // TODO: Check if we have consent to display this tool
-  return true;
-};
+const SocialContentWrapper = ({ align = 'center', tool, children, data }) => {
+  const CheckPrivacyConsent = config.getComponent('CheckPrivacyConsent')
+    .component;
+
+  const SocialContentProviders = {
```

Comment:
nitpick: I would move this up above SocialContentWrapper so that it is defined at the module level. That way the object is created once and reused, instead of recreated each time SocialContentWrapper is called.